src/PersonObjects/IPlayer.ts

```diff
@@ -248,7 +248,7 @@ export interface IPlayer {
   queryStatFromString(str: string): number;
   getIntelligenceBonus(weight: number): number;
   getCasinoWinnings(): number;
-  quitJob(company: string): void;
+  quitJob(company: string, sing?: boolean): void;
```

Comment:
Whats the `sing` param do?

Reply:
nothing atm, its more of a "just incase" basically if the func ever needed to do logging it would be needed